broker-cli/proto/broker.proto

```diff
@@ -149,6 +149,11 @@ message GetIdentityResponse {
   string public_key = 1;
 }
 
+message RegisterResponse {
+  // The entity_id of the entity registered with the authorization service
```

Comment:
The Broker probably shouldn't have a notion of the authorization service, I would just say that this is "the Identifier assigned by the Relayer for the Entity associated with the provided Public Key".